### PR TITLE
Fix zsh tab cause current input to be duplicated bug

### PR DESCRIPTION
**Pull request by Azhng**
Turns out to be an issue with locales.
Other minor fixes for running shell scripts from ruby scripts

### CODE REVIEW

File: bin/frun.rb [hunk 1]
```diff
@@ -18,5 +18,5 @@
```
Comment (arora-aman) [inline bin/frun.rb]: This goes away with #41

File: bin/fk.rb [hunk 2]
```diff
@@ -19,5 +19,5 @@
 
 cmd = cmds[action]["run"]
 
-puts "fk is running #{cmd}"
-system(cmd)
+puts "fk is running #{cmd} #{ARGV[2..-1].join(' ')}"
+exec([cmd, ARGV[2..-1].join(' ')].join(' '))
```
Comment (arora-aman) [inline bin/fk.rb]: Why the extra join at the end?

**Review by arora-aman — COMMENTED** [review]
Same question at #41, what is `az.exec` for

**Comment by Azhng** [comment]
`az.exec` was original to replace the `system()` in `frun.rb` with `exec()`. Then I noticed the tab causing duplicated input characters in Docker with zsh so I fixed along the way. I should have probably made those into separate PRs but those fixes were just too small so ¯\_(ツ)_/¯

File: bin/fk.rb (hunk 2 above)
Comment (Azhng) [reply bin/fk.rb]: the first `join()` was to join all the arguments to the `cmd` into a single string then the second join is to join the original `cmd` with the `args` string. Refactored out the args to make it more clear.

File: bin/frun.rb (hunk 1 above)
Comment (Azhng) [reply bin/frun.rb]: :+1:

**Comment by Azhng** [comment]
Going rogue

**Pull request merged**